bluemira/plasma_physics/collisions.py

```diff
@@ -0,0 +1,180 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2022 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+
+"""
+Tokamak plasma collision formulae.
+"""
+
+import numpy as np
+
+from bluemira.base.constants import (
+    ELECTRON_MASS,
+    EPS_0,
+    EV_TO_J,
+    H_PLANCK,
+    K_BOLTZMANN,
+    PROTON_MASS,
+)
+
+
+def debye_length(temperature, density):
+    """
+    Debye length
+
+    Parameters
+    ----------
+    temperature: float
+        Temperature [K]
+    density: float
+        Density [m^-3]
+
+    Returns
+    -------
+    debye_length: float
+        Debye length [m]
+    """
+    return np.sqrt(EPS_0 * K_BOLTZMANN * temperature / (EV_TO_J**2 * density))
+
+
+def reduced_mass(mass_1, mass_2):
+    """
+    Calculate the reduced mass of a two-particle system
+
+    Parameters
+    ----------
+    mass_1: float
+        Mass of the first particle
+    mass_2: float
+        Mass of the second particle
+
+    Returns
+    -------
+    mu_12: float
+        Reduced mass
+    """
+    return (mass_1 * mass_2) / (mass_1 + mass_2)
+
+
+def thermal_velocity(temperature, mass):
+    """
+    Parameters
+    ----------
+    temperature: float
+        Temperature [K]
```

Comment:
Just realised, temperatures in public functions [should be in Celsius](https://bluemira.readthedocs.io/en/develop/conventions.html#unit-convention).

Reply:
Uff.. yeah I knew I would come to regret that decision :D. It feels extremely unnatural to use Celsius in plasma physics functions like this. It's already weird we're not using [eV] or [keV]. To be honest, [K] is SI... and [C] is like an accepted derivate. Perhaps we should be simple and use [K] everywhere. It does seem a natural middle ground between plasma physics [eV] and engineering [C].

Thoughts @hsaunders1904 @je-cook @ivanmaione ?

(Incidentally I know that we could use parameters here or pint to get the conversions more naturally, but I felt that would be overkill here, and certainly inconsistent with the rest of the stuff we presently have).

Reply:
Kelvin, to me, does seem the natural choice, seeing as we use SI for almost everything else. However, we'd obviously have to consider how difficult it would be to update the existing temperatures throughout the code (it could well be trivial, but may well not be).

In any case, this is a question that's out of the scope of this PR. My thinking is that we should try to be consistent in the first instance. I say get this PR merged using Celsius here for now, and we can revisit the question in a weekly meeting or at the workshop.

Reply:
I prefer Kelvin too and I think we should just stomach the changes that may need to be made. We shouldnt be hardcoding many temperatures anyway and if we are we can just use the unit converter function.

Maybe for now just stick in a 
```
raw_uc(temperature, 'celsius', 'kelvin')
```
and we can remove it later